src/stories/Library/material-grid/init-material-grid.js

```diff
@@ -0,0 +1,20 @@
+window.addEventListener("load", () => {
```

Comment:
I'm not sure if it's necessary but maybe we should use a DOMContentLoaded here

Reply:
I'm not sure if everything is loaded at that time once used in the drupal setup, but we can test it with DOMContent first and change it later if it's a problem. 👍